src/controllers/processors/SessionCleanupProcessor.ts

```diff
@@ -0,0 +1,41 @@
+import { Session } from 'ch-node-session-handler';
+import { Request } from 'express';
+import { provide } from 'inversify-binding-decorators';
+
+import { FormActionProcessor } from 'app/controllers/processors/FormActionProcessor';
+import { loggerInstance } from 'app/middleware/Logger';
+import { ApplicationData, APPLICATION_DATA_KEY } from 'app/models/ApplicationData';
+import { APPLICATION_DATA_UNDEFINED, SESSION_NOT_FOUND_ERROR } from 'app/utils/CommonErrors';
+import { CONFIRMATION_PAGE_URI } from 'app/utils/Paths';
+
+@provide(SessionCleanupProcessor)
+export class SessionCleanupProcessor implements FormActionProcessor {
+
+    public async process(req: Request): Promise<void> {
```

Comment:
I think we need some unit tests for this processor

Reply:
I don't think we need that since all the logic was tested implicitly by the CheckYourAppealController.test.ts.